pkg/ccl/streamingccl/streamclient/partitioned_stream_client.go

```diff
@@ -358,3 +377,50 @@ func (p *partitionedStreamSubscription) Events() <-chan streamingccl.Event {
 func (p *partitionedStreamSubscription) Err() error {
 	return p.err
 }
+
+func URIWithTLSCertsRemoved(remote *url.URL) *url.URL {
+	ret := *remote
+	v := ret.Query()
+	v.Del("sslcert")
+	v.Del("sslkey")
+	v.Del("sslrootcert")
+	v.Del("sslinline")
+	ret.RawQuery = v.Encode()
+	return &ret
+}
+
+func AddTLSCertsToConfigFromInlineURL(remote *url.URL, tlsConfig *tls.Config) error {
```

Comment:
I'm guessing there's some expected future use of these outside the package requiring them to be exported?

Reply:
Nope, I originally was going to have this live in some utils package, but then decided to just put it here. Thanks for noting it.